Gemfile

```diff
@@ -29,6 +29,9 @@ gem 'pvb-instrumentation',
   git: 'https://github.com/ministryofjustice/pvb-instrumentation.git',
   ref: 'a264627211f2bf68f4e388200b2a050fe9081504'
 
+gem 'email_address_validation',
+  git: 'https://github.com/ministryofjustice/email_address_validation'
```

Comment:
@KPobeeNorris just noticing this now. Can you create a tag like 'v0.1.0' and use point to this tag or at least reference the the commit using the `ref` keyword just like above.
That would ensure we use a deterministic version of the gem.

Reply:
after that I think It's all good to go.